roles/migrid_logstash_gdp/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+elastic_release: 8
```

Comment:
All vars should be prefixed, usually with the role name but since that might be long, maybe `migrid_` is enough here.
Other variables from this collection all use the `migrid_` prefix.
Without prefix, variable like these might interfere with other roles.